fastlane/lanes/build.rb

```diff
@@ -212,7 +219,7 @@
       derived_data_path: DERIVED_DATA_PATH,
       export_team_id: get_required_env('INT_EXPORT_TEAM_ID'),
       export_method: 'enterprise',
-      export_options: { method: 'enterprise' }
+      export_options: { **COMMON_EXPORT_OPTIONS, method: 'enterprise' }
```

Comment:
I mentioned the issue is only in `method: 'app-store'` builds. However, it still feels appropriate to apply the fix to all builds.